get_alphabetic_letters.py

```diff
@@ -1,4 +1,7 @@
 def get_letters():
+    """
+    :return: list of all lower case letters and upper case
+    """
     lower_case = [chr(letter) for letter in range(ord('a'), ord('z') + 1)]
     upper_case = [chr(letter) for letter in range(ord('A'), ord('Z') + 1)]
     return lower_case, upper_case
```

Comment:
You wrote that you are returning a list but instead you have returned a tuple of lists